substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/util/LLVMIRBuilder.java

```diff
@@ -91,12 +92,16 @@ public void setMainFunction(String functionName, LLVMTypeRef functionType) {
     /* Module */
 
     public byte[] getBitcode() {
+        // LLVM.LLVMDumpModule(module);
+
         LLVMMemoryBufferRef buffer = LLVM.LLVMWriteBitcodeToMemoryBuffer(module);
         BytePointer start = LLVM.LLVMGetBufferStart(buffer);
         int size = NumUtil.safeToInt(LLVM.LLVMGetBufferSize(buffer));
 
         byte[] bitcode = new byte[size];
         start.get(bitcode, 0, size);
+        start.close();
```

Comment:
I'd prefer to see this as an auto-closing `try` block

Reply:
Done.